tests/foreman/api/test_syncplan.py

```diff
@@ -106,20 +108,21 @@ def test_positive_get_routes(self):
         )
         for response in (response1, response2):
             response.raise_for_status()
-        self.assertEqual(response1.json()['results'], response2.json()['results'])
+        assert response1.json()['results'] == response2.json()['results']
 
 
-class SyncPlanCreateTestCase(APITestCase):
+@pytest.mark.usefixtures("setup_org_for_syncplan")
+class TestSyncPlanCreate:
     """Tests specific to creating new sync plans."""
 
-    @classmethod
-    def setUpClass(cls):
+    @pytest.fixture(scope='class')
+    def setup_org_for_syncplan(self):
         """Create an organization which can be re-used in tests."""
-        super(SyncPlanCreateTestCase, cls).setUpClass()
-        cls.org = entities.Organization().create()
+        TestSyncPlanCreate.org = entities.Organization().create()
 
+    @pytest.mark.parametrize("enabled", [False, True])
     @tier1
-    def test_positive_create_enabled_disabled(self):
+    def test_positive_create_enabled_disabled(self, enabled):
```

Comment:
Add the setup_org_for_syncplan fixture in each test case where we are using the organization.

```
@tier1
@pytest.mark.parametrize("enabled", [False, True])
def test_positive_create_enabled_disabled(self, enabled, setup_org_for_syncplan):
     sync_plan = entities.SyncPlan(enabled=enabled,organization=setup_org_for_syncplan).create()
     sync_plan = sync_plan.read()
     assert sync_plan.enabled == enabled
```

Reply:
@devendra104 , please review again because I have made changes while removing the classes, and all tests now use `module_org` fixture.  Thank you